src/lib/reduxStore.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @file reduxStore.js
+ * Constructs Redux store and persistor.
+ */
+
+import { createStore, applyMiddleware } from 'redux';
+import { persistStore, persistReducer } from 'redux-persist';
+import storage from 'redux-persist/lib/storage';
+import thunkMiddleware from 'redux-thunk';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import reducer from '../reducers';
+
+const persistedReducer = persistReducer(
```

Comment:
what is the persistor for? are we storing user session state somewhere for debugging or something?

Reply:
@infiniteluke This is primarily for user session state, we need the accessToken, refreshToken, token expiration info, etc to make authenticated requests.

Reply:
I am thinking, perhaps non-user data shouldn't be persistent. What do you think? Is there a way to configure persistence for one reducer?